### PR TITLE
Test with multibucket config

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -96,6 +102,16 @@ pipeline:
       - wait-for-it scality:8000
       - cd /drone/server
       - php ./occ s3:create-bucket owncloud --accept-warning
+      - php ./occ s3:create-bucket owncloud0 --accept-warning
```
Comment (PVince81) [inline .drone.yml]: `for I in $(seq 1 9); do ... owncloud$I ...; done` ;-)

File: .drone.yml [hunk 2]
```diff
@@ -380,7 +396,14 @@ matrix:
       PHP_VERSION: 7.2
       TEST_SUITE: phpunit
       CORE_BRANCH: stable10
-    # 7.1
+
+    - STORAGE: scality
+      STORAGE_CONFIG_SUFFIX: .multibucket
+      PHP_VERSION: 7.1
+      TEST_SUITE: phpunit
+      CORE_BRANCH: stable10-cca67d6
```
Comment (PVince81) [inline .drone.yml]: ok so we'll wait for merge

**Comment by DeepDiver1975** [comment]
@PVince81 all addressed ...

**Comment by DeepDiver1975** [comment]
@PVince81 I'll merge this now - all core branches are merged and :robot: is green

**Comment by mmattel** [comment]
@DeepDiver1975 is this documentation relevant?

**Comment by DeepDiver1975** [comment]
> @DeepDiver1975 is this documentation relevant?

multibucket is an old feature which should be documented already properly (with the note that we {want} deprecate it) ....

**Pull request merged**